config/env/default.js

```diff
@@ -9,6 +9,9 @@ module.exports = {
   },
   port: process.env.PORT || 3000,
   host: process.env.HOST || '0.0.0.0',
+  // DOMAIN config should be set to the fully qualified application accessible
+  // URL. For example: https://www.myapp.com (including port if required).
+  domain: process.env.DOMAIN,
```

Comment:
If not set, should this default to localhost? Or at least default to localhost in the development config?